content/intro-to-storybook/vue/fr/contribute.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Contribuer'
+description: 'Aider à partager Storybook dans le monde'
+---
+
+Les contributions à Learn Storybook sont encouragées ! S'il s'agit de petites corrections comme de la grammaire our de la ponctuation, ouvrez une pull request. S'il s'agit d'importants changements, [ajouter une issue](https://github.com/chromaui/learnstorybook.com/issues) pour échanger.
+
+Learn Storybook a été principalement créé et maintenu par la communauté, nous avons donc besoin de l'aide de tous pour le garder à jour et nous assurer que <TODO>and ensure any rough edges are smoothed out over time.</TODO> Toute aide est appréciée !
```

Comment:
Translation not finished here :)

Learn Storybook a été principalement créé et maintenu par la communauté, nous avons donc besoin de l'aide de tous pour le garder à jour et nous assurer que ...
les imperfections seront corrigés avec le temps.

Reply:
outch, sorry for this ! it's changed